ebos/eclwriter.hh

```diff
@@ -349,14 +351,27 @@ public:
         Inplace inplace;
         {
             OPM_TIMEBLOCK(outputFipLogAndFipresvLog);
-            inplace = eclOutputModule_->outputFipLog(miscSummaryData, regionData, 0,
+            
+            const std::string version = moduleVersionName();
+
+            auto es_ = simulator_.vanguard().eclState();
+            auto sched_ = simulator_.vanguard().schedule();
+
+            boost::posix_time::ptime start_time = boost::posix_time::from_time_t(sched_.getStartTime());
```

Comment:
Since we use the `sched_` "only" to access the start time, maybe we can write
```C++
boost::posix_time::ptime start_time = boost::posix_time::from_time_t(simulator_.vanguard(),schedule().getStartTime());
```
to avoid creating a full copy of the `Schedule` object?